lib/dialects/mssql/query.js

```diff
@@ -67,16 +67,23 @@ class Query extends AbstractQuery {
           } else {
             resolve(this.formatResults());
           }
-        });
+        }, this.options.transaction.name);
+      } else if (_.includes(this.sql, 'SAVE TRANSACTION')) {
+        connection.saveTransaction(err => {
+          if (!!err) {
```

Comment:
@felixfbecker `beginTransaction`, `rollbackTransaction` and `commitTransaction` also has boolean cast. Should i remove them aswell?